base/iterator.jl

```diff
@@ -284,19 +284,23 @@ end
 
 done(it::Cycle, state) = state[2]
 
-# Repeat an object infinitely many times
+
+# Repeated - repeat an object `n` or infinitely many times
```

Comment:
the type is only for infinite repeats, the lower case method happens to have a convenience method but that will return a Take object wrapping a Repeated